### DISCUSSION TITLE
Cmake errors when building project

### DISCUSSION THREAD

**Discussion by CodeLikeCXK** [o3de/o3de#1950] [Help Wanted]
CMake Error at CMakeLists.txt:8 (ly_get_list_relative_pal_filename):
  Unknown CMake command "ly_get_list_relative_pal_filename".

Also how to add configuration for building in CLion?

**Comment by amznestebanpapp** [comment]
Can you post how you are building? that looks like you started from a CMakeLists.txt different than the one from the root

Regarding CLion, supporting, CMake doesnt generate for CLion, CLion has to understand cmake or some other generator. If CLion is not able to understand CMake here (because our implementation is complex), my recommendation would be to generate nmake and see if that opens up in CLion.
